### ISSUE TITLE
How to draw VOC dataset pseudo mask?

### ISSUE THREAD

**Issue by vb123er951** [HSG-AIML/TokenMasking-WSSS#3]
I am trying to use your work, on the VOC dataset.
I saw the voc option in training script, so training is fine. 
But in the `generate_pseudomask.py` the only accepted datasets are dfc and ade20k.
How do you draw the pseudo mask of VOC dataset, as showed in the paper?
Thanks.
